src/objective-c/GRPCClient/private/GRPCWrappedCall.h

```diff
@@ -75,6 +75,7 @@
 @interface GRPCWrappedCall : NSObject
 
 - (instancetype)initWithHost:(NSString *)host
+                  serverName:(nullable NSString *)serverName
```

Comment:
Specifying nullability for only partial parameters here seems to break compile in Xcode 8.x. Since it is internal interface, I think it is probably ok to just remove the `nullable` specifier. Same for `grpchost.{h,m}`